packages/core/admin/admin/src/contexts/themeToggle.ts

```diff
@@ -0,0 +1,19 @@
+import { createContext } from 'react';
+
+import { StrapiTheme } from '@strapi/design-system';
+
+type ThemeName = 'light' | 'dark';
+
+interface ThemeToggleContextContextValue {
+  currentTheme?: ThemeName;
+  onChangeTheme?: (nextTheme: ThemeName) => void;
+  themes?: {
+    dark: StrapiTheme;
+    light: StrapiTheme;
+  };
+}
+
+export const ThemeToggleContext = createContext<ThemeToggleContextContextValue>({
+  currentTheme: undefined,
+  themes: undefined,
```

Comment:
do we need to declare them as undefined?

Reply:
No I've just added them for readability, but I agree it is probably better to remove them.